//
// PROJECT:         Aspia
// FILE:            category/category_open_files.proto
// LICENSE:         GNU Lesser General Public License 2.1
// PROGRAMMERS:     Dmitry Chapyshev (dmitry@aspia.ru)
//

syntax = "proto3";

option optimize_for = LITE_RUNTIME;

package aspia.proto;

message OpenFile
{
    message Item
    {
        uint32 id         = 1;
        string user_name  = 2;
        uint32 lock_count = 3;
        string file_path  = 4;
    }

    repeated Item item = 1;
}
